Teams.md

```diff
@@ -4,34 +4,34 @@
 
 ### Co-Team Leads(Maintainers)
 
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
 
 ### Contributors
 
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
 
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
 
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
+- First Name  Last Name  Preferred Name  GitHub username  location timezone
 
 ## Americas01
 
 ---
 
 ### Co-Team Leads(Maintainers)
 
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
+- [Chrissy Belzie](https://www.linkedin.com/in/christinebelzie/)  Preferred Name  [ChrissyCodes](https://github.com/CBID2)  [christinebelzie@gmail.com](mailto:christinebelzie@gmail.com)  location  timezone
```

Comment:
```suggestion
- [Christine Belzie](https://www.linkedin.com/in/christinebelzie/)  Chrissy
[ChrissyCodes](https://github.com/CBID2)  [christinebelzie@gmail.com](mailto:christinebelzie@gmail.com)  NY  EST
```

Reply:
Thanks! Updated now. I left preferred names and a couple of the timezones as the default for now they're currently unknown based off the spreadsheet information